source/util/bitutils.h

```diff
@@ -27,6 +27,8 @@
 #ifndef LIBSPIRV_UTIL_BITUTILS_H_
 #define LIBSPIRV_UTIL_BITUTILS_H_
 
+#include <stdint.h>
```

Comment:
I guess this is a symptom of VS 12?
In C++ code like this we'd prefer #include <cstdint>